package.json

```diff
@@ -6,6 +6,12 @@
   "main": "dist/cjs/index.js",
   "module": "dist/pkg/index.js",
   "types": "dist/pkg/index.d.ts",
+  "browser": {
+    "./dist/cjs/node": "dist/cjs/node.browser.js",
+    "./dist/cjs/node.js": "dist/cjs/node.browser.js",
+    "./dist/pkg/node.js": "dist/pkg/node.browser.js",
+    "./dist/pkg/node": "./dist/pkg/node.browser.js"
```

Comment:
This last entry is the only one actually needed by `vite`, but I've worked with some bundlers that require the extension, so covering all bases here.